bin/run

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
I would, but the convention is to use `python setup.py` as a task runner. Try `python setup.py - -
help-commands` from the project root sometime.

As for using a shell script instead of a Makefile, I wanted to defer all unknown commands to `python setup.py`. I also know how foreign the Makefile syntax is to some people and figured this would be more community-friendly.